include/ceed/ceed.h

```diff
@@ -220,8 +220,6 @@ CEED_EXTERN const CeedVector CEED_VECTOR_NONE;
 /// @ingroup CeedBasis
 CEED_EXTERN const CeedBasis CEED_BASIS_NONE;
 
-CEED_EXTERN const CeedBasis CEED_BASIS_COLLOCATED;
-
```

Comment:
It's late now, but I think we could have deprecated this initially. https://gcc.gnu.org/onlinedocs/gcc/Common-Variable-Attributes.html#index-deprecated-variable-attribute

Mohadese just encountered use of the old name in RDycore, which has been building without warnings for those using release libCEED via PETSc (even though `CEED_BASIS_COLLOCATED` was documented as deprecated by then).